WooCommerce/Classes/ViewModels/Order Details/Refund Details/RefundedOrderItemCellViewModel.swift

```diff
@@ -0,0 +1,108 @@
+import Foundation
+import Yosemite
+
+
+// MARK: - View Model for a refunded order item cell
+//
+struct RefundedOrderItemCellViewModel {
```

Comment:
similar to the previous comment, it looks like this struct only has to expose `imageURL`, `name`, `quantity`, `price`, and `sku`. could you make sure the other properties are private?